src/stores/messageHistoryStore/messageHistoryStore.ts

```diff
@@ -12,6 +12,10 @@ export class MessageHistoryStore {
     }
   }
 
+  public getMessages = () => {
```

Comment:
could be replaced with `getSnapshot()` ? or removed if the debug button is removed (or fine to keep too)